Automattic/jetpack
# Blog Stats Widget: display message when no stats are displayed

For newly created sites, there's no Stats to be displayed in the Stats widgets, however the current message is misleading. The message suggests that there's an error with the widget.


This PR adds a new state to the widget, clarifying that there's nothing wrong, it's just that there is no stats to be displayed yet.

Before:
<img width="231" alt="screen-shot-on-2021-04-09-at-12-28-21" src="https://user-images.githubusercontent.com/971483/114233135-48f0ad80-9953-11eb-9825-a56cfa0d5e0f.png">

After:
![Captura de Tela 2021-04-09 às 16 51 38](https://user-images.githubusercontent.com/971483/114233585-f237a380-9953-11eb-8eec-4df63a26a79c.png)



#### Changes proposed in this Pull Request:

* Display a proper message in the Stats widget for new sites without stats

#### Jetpack product discussion


p9dueE-2EQ-p2#comment-4807

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Launch a new JN site
* Connect it
* Jetpack > Settings > Writing
* Enable widgets
* Go to Appearence > Widgets
* Add a Blog Stats widget
* Check the front end of the site and see the new message

If possible, also test this branch with an existing site with some stats to make sure the widget is still working as expected.
